README/ReleaseNotes/v628/index.md

```diff
@@ -557,6 +567,8 @@ accessing widget via configured ssh tunnel.
 
 ## Language Bindings
 
+Python 3.10 is now supported.
```

Comment:
I believe we already supported 3.10 and we now also support 3.11

Reply:
Updated to 3.11!